nebula-components/settings/components/_tabs.scss

```diff
@@ -0,0 +1,16 @@
+$nb-tabs-background: $nb-color-white !default;
+
+$nb-tab-padding: $nb-spacing-unit !default;
+$nb-tab-color: $nb-color-grey-v-dark !default;
+$nb-tab-background: $nb-color-grey-v-light !default;
+$nb-tab-transition: background-color 0.5s ease !default;
+
+$nb-tab-color-disabled: $nb-color-grey !default;
+$nb-tab-background-disabled: $nb-color-grey-v-light !default;
+
+$nb-tab-color-active: inherit !default;
+$nb-tab-background-active: $nb-color-white !default;
+$nb-tab-border-active: 5px solid $nb-color-alpha !default;
```

Comment:
Use global variables:
```
$nb-active-border-width: [value from navbar] !default;
$nb-active-border-color: $nb-color-alpha !default;
$nb-active-border: $nb-active-border-width solid $nb-active-border-color !default;
``` 
Consume these variables in `navbar` `status-card` `tabs`